File: csharp/create_calendar_from_calendargroup.cs
var client = new RestClient("https://graph.microsoft.com/v1.0/me/calendarGroups/%7Bid%7D/calendars");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{ }", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);

File: python/create_calendar_from_calendargroup.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("POST", "/v1.0/me/calendarGroups/%7Bid%7D/calendars", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
